src/python/pants/backend/python/dependency_inference/parse_python_imports_test.py

```diff
@@ -94,24 +108,39 @@ def test_normal_imports(rule_runner: RuleRunner) -> None:
             import subprocess23 as subprocess
 
         __import__("pkg_resources")
+        __import__("dunder_import_ignored")  # pants: ignore
+        __import__(  # pants: ignore
+            "not_ignored_but_looks_like_it_could_be"
+        )
+        __import__(
+            "ignored"  # pants: ignore
+        )
+        __import__(  # pants: ignore
+            "also_not_ignored_but_looks_like_it_could_be"
```

Comment:
How is this case different from
```
__import__(  # pants: ignore
    "not_ignored_but_looks_like_it_could_be"
)
```

Reply:
Oh whoops, supposed to be on the end-parenthesis. Good catch!